src/app/demo/demo.component.html

```diff
@@ -1,6 +1,33 @@
 <div class="demo-page">
   <div class="nav-side-menu">
-    <div class="brand">angular-archwizard</div>
+    <div class="brand">
+      <a class="navbar-brand text-white" href="#">
+        <img src="https://angular.io/assets/images/logos/angular/angular.svg" width="32px" height="32px"/>
+        angular-archwizard
+      </a>
+
+      <div class="btn-group btn-group-toggle d-block mx-auto dir" data-toggle="buttons">
+        <label
+          class="btn btn-secondary"
+          (click)="changeDirTo('ltr')">
+          <input type="radio" name="options" id="ltr" checked> LTR
+        </label>
+        <label
+          class="btn btn-secondary"
+          (click)="changeDirTo('rtl')">
+          <input type="radio" name="options" id="rtl"> RTL
+        </label>
+      </div>
```

Comment:
This looks a bit strange in (my) Firefox:

![grafik](https://user-images.githubusercontent.com/18488086/79699407-beb9c000-828f-11ea-8c19-79141a55c29a.png)

Chromium in comparison:

![grafik](https://user-images.githubusercontent.com/18488086/79699410-cc6f4580-828f-11ea-822a-cffab7342374.png)

I think the newline should not be there right?

Reply:
yes, that a bug. I have removed the fixed width and replaced it with text-align: center. and it fixed this issue.